content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -1,1440 +1,1363 @@
+* [Portenta X8 Pinout](https://docs.arduino.cc/static/019dd9ac3b08f48192dcb1291d37aab9/ABX00049-full-pinout.pdf)
+
+### Datasheet
+
+The full datasheet is available and downloadable as PDF from the link below:
+* [Portenta X8 Datasheet](https://docs.arduino.cc/resources/datasheets/ABX00049-datasheet.pdf)
+
+### Schematics
+
+The full schematics are available and downloadable as PDF from the link below:
+* [Portenta X8 Schematics](https://docs.arduino.cc/resources/schematics/ABX00049-schematics.pdf)
+
+### STEP Files
+
+The full _STEP_ files are available and downloadable from the link below:
+* [Portenta X8 STEP files](../../downloads/ABX00049-step.zip)
+
+### Features Overview
+
+Portenta X8 integrates two main programming experiences: the **Yocto Linux** and popular **Arduino** environments.
+
+![Portenta X8 Features Overview](assets/portenta-x8-functionality-overview.png "Portenta X8 Features Overview")
+
+To explore specific sections in more detail, please click on the links below that interest you:
+
+* [Linux Environment](#linux-environment)
+* [Arduino Environment](#arduino-environment)
+* [Arduino Cloud](#portenta-x8-with-arduino-cloud)
+
+### Portenta X8: Linux & Arduino Interoperability
+
+The Portenta X8 runs a **Yocto** based Linux distribution. It allows users to upload sketches to the M4 core of the STM32H7 using the Arduino IDE. The process of uploading sketches may seem familiar to Arduino users, but on Portenta X8, the system uses a service called `monitor-m4-elf-file.service`.
+
+This service monitors a specific directory for updated sketches and flashes the M4 core with the new code via OpenOCD. This feature enables seamless integration between the Linux and Arduino environments on the board.
+
+The Portenta X8 employs **Remote Procedure Call (RPC)** mechanisms to enable communication between the Linux system and the Arduino cores. RPC allows the two processors (M4 and M7 cores) to exchange data, with the M7 core handling data transfers between the M4 core running Arduino sketches and the Linux environment.
+
+The communication uses **MessagePack-RPC**, a lightweight protocol for serialized data transfer, supporting various communication protocols like OpenAMP, SPI, and TCP/IP. This setup ensures efficient interaction between the Linux side and Arduino sketches without the complexity of traditional dual-core processing.
+
+The Portenta X8 offers a powerful, versatile platform by combining Linux capabilities with Arduino’s ease of use. Users can deploy and manage device-independent applications using Docker containers while also taking advantage of Arduino’s rich ecosystem for embedded development. This hybrid environment supports flexible development, making it ideal for complex, multifunctional projects.
+
+***For more in-depth details on fundamentals of the Portenta X8, please refer to this [documentation](https://docs.arduino.cc/tutorials/portenta-x8/x8-fundamentals/).***
+
+## Portenta X8 OS Image Update
```

Comment:
I'd recommend moving this section after the section named Arduino Linux Wizard Homepage

Reply:
Updated!